components/serializer.rst

```diff
@@ -913,6 +913,40 @@ These are the options available:
 ``remove_empty_tags``
     If set to true, removes all empty tags in the generated XML.
 
+Handling Constructor Arguments
+------------------------------
+
+If the constructor of a class defines arguments, as usually happens with
```

Comment:
It's not exact: the serializer deals well with constructor arguments, if they are present in the object to deserialize.

The `default_constructor_arguments` is useful only when the value has not been provided in the data to deserialize.